web/src/components/molecules/Workspace/types.ts

```diff
@@ -33,6 +33,10 @@ export interface MemberInput {
 }
 
 export type Role = "WRITER" | "READER" | "MAINTAINER" | "OWNER";
+// t("WRITER");
+// t("READER");
+// t("MAINTAINER");
+// t("OWNER");
```

Comment:
also

Reply:
There is no need to comment these out, you can do this:
```
t("WRITER");
t("READER");
t("MAINTAINER");
t("OWNER");
```